ui/app/controllers/jobs/job/index.js

```diff
@@ -62,4 +66,62 @@ export default class IndexController extends Controller.extend(
   setStatusMode(mode) {
     this.statusMode = mode;
   }
+
+  @tracked
+  childJobsController = new AbortController();
+
+  childJobsQuery(params) {
+    this.childJobsController.abort();
+    this.childJobsController = new AbortController();
+
+    return this.store
+      .query('job', params, {
+        adapterOptions: {
+          abortController: this.childJobsController,
+        },
+      })
+      .catch((e) => {
+        if (e.name !== 'AbortError') {
+          console.log('error fetching job ids', e);
+        }
+        return;
+      });
+  }
+
+  @tracked childJobs = [];
+
+  resetQueryIndex({ id, namespace }) {
+    this.watchList.setIndexFor(`child-jobs-for-${id}-${namespace}`, 1);
+  }
+
+  @restartableTask *watchChildJobs(
+    { id, namespace },
+    throttle = Ember.testing ? 0 : 2000
+  ) {
+    while (true) {
+      let params = {
+        filter: `ParentID == "${id}"`,
+        namespace,
+        include_children: true,
+      };
+      params.index = this.watchList.getIndexFor(
+        `child-jobs-for-${id}-${namespace}`
+      );
+
+      const childJobs = yield this.childJobsQuery(params);
+      if (childJobs) {
+        if (childJobs.meta.index) {
+          this.watchList.setIndexFor(
+            `child-jobs-for-${id}-${namespace}`,
+            childJobs.meta.index
+          );
+        }
+        this.childJobs = childJobs;
+        yield timeout(throttle);
+      }
+      if (Ember.testing) {
+        break;
+      }
+    }
+  }
```

Comment:
Generally follows the main "duelling queries" pattern from the jobs index list of having the route's `setupController` kick off a restartable ember concurrency task, rather than use `@watchQuery`, which has some "when do I invalidate the cache" issues.